public/scripts/plugins/pcr/templates/pcr_product_view.hbs

```diff
@@ -75,9 +81,71 @@
 <br>
 {{/if}}
 
+<br />
+
+
+
+<h5>Optimized parameters</h5>
```

Comment:
@DesireeSy I think we want to call this "Optimized RDP PCR Parameters".  What do you think?